beacon_node/http_api/src/lib.rs

```diff
@@ -1532,6 +1532,53 @@ pub fn serve<T: BeaconChainTypes>(
             },
         );
 
+    // GET beacon/deposit_snapshot
+    let get_beacon_deposit_snapshot = eth_v1
+        .and(warp::path("beacon"))
+        .and(warp::path("deposit_snapshot"))
+        .and(warp::path::end())
+        .and(warp::header::optional::<api_types::Accept>("accept"))
+        .and(eth1_service_filter.clone())
+        .and_then(
+            |accept_header: Option<api_types::Accept>, eth1_service: eth1::Service| {
+                blocking_task(move || match accept_header {
+                    Some(api_types::Accept::Json) => {
```

Comment:
I think the default should be JSON if no header is specified
```suggestion
                    Some(api_types::Accept::Json) | None => {
```